In-house patch to add the 'solariszones' and 'ldoms' hypervisors to
Nova's list of known hypervisors. This patch has not yet been submitted
upstream.

--- nova-13.1.0/nova/compute/hv_type.py.~1~	2016-06-14 08:45:36.000000000 -0700
+++ nova-13.1.0/nova/compute/hv_type.py	2016-08-05 11:46:42.648351250 -0700
@@ -42,6 +42,8 @@ PARALLELS = "parallels"
 VIRTUOZZO = "vz"
 PHYP = "phyp"
 QEMU = "qemu"
+SOLARISZONES = "solariszones"
+LDOMS = "ldoms"
 TEST = "test"
 UML = "uml"
 VBOX = "vbox"
@@ -64,6 +66,8 @@ ALL = (
     PARALLELS,
     PHYP,
     QEMU,
+    SOLARISZONES,
+    LDOMS, 
     TEST,
     UML,
     VBOX,
